numpy/polynomial/chebyshev.py

```diff
@@ -1012,8 +1012,8 @@ def chebint(c, m=1, k=[], lbnd=0, scl=1, axis=0):
     Raises
     ------
     ValueError
-        If ``m < 1``, ``len(k) > m``, ``np.isscalar(lbnd) == False``, or
-        ``np.isscalar(scl) == False``.
+        If ``m < 1``, ``len(k) > m``, ``np.ndim(lbnd) != 0``, or
+         ``np.ndim(scl) != 0``.
```

Comment:
Hmm, looking at the code, I notice that it is not checked that `lbnd` is scalar for any of the polynomial types. Seems to have slipped by somehow ...

The suggested check should work.

Reply:
If you want to make the fix, go ahead, otherwise I'll do it after this goes in.

Reply:
I think we get away with it because there as `coeffs[i] = lbd` assignment at some point, that enforces the constraint. There's no such test for `scl` - indeed, you can pass an array and get a (probably meaningless) result

Reply:
I'll let you make the fix - I'm just trying to eliminate occurrences of `isscalar`

Reply:
OK.